pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -329,11 +329,21 @@ func ValidateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 	causes = append(causes, ValidateVirtualMachineInstanceMetadata(field.Child("template", "metadata"), &spec.Template.ObjectMeta, config, accountName)...)
 	causes = append(causes, ValidateVirtualMachineInstanceSpec(field.Child("template", "spec"), &spec.Template.Spec, config)...)
 
+	validateDataVolumeTemplate(field, spec, &causes)
```

Comment:
Any reason for passing `&causes` instead of just calling `append()` as we do above?

It's a widely used pattern in these webhooks and while passing `&causes` likely reads a little better we should likely stick to the existing pattern in this partial refactor of `ValidateVirtualMachineSpec`.

Reply:
Good point, I've updated it to use `append` rather than `causes`